app/index.js

```diff
@@ -309,6 +310,16 @@ app.on('ready', () => {
       }
     }));
 
+    // If we're on Mac make a Dock Menu
+    if (process.platform === 'darwin') {
+      const {app, Menu} = require('electron');
```

Comment:
The linter didn't complain no, I think the rule it enforces it that you keep it consistent throughout that part.